jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/PollMessagePane.kt

```diff
@@ -0,0 +1,9 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.services.sqs.toolwindow
+
+import javax.swing.JPanel
+
+class PollMessagePane {
+    var component = JPanel()
```

Comment:
Ah this should be bound from the form file, checkout `CreateBucketPanel` as an example